pdal/private/gdal/Raster.cpp

```diff
@@ -32,8 +32,11 @@
 * OF SUCH DAMAGE.
 ****************************************************************************/
 
+#pragma warning(push)
```

Comment:
Rather than sprinkling this all over the place, maybe we should make our own private GDAL header that does this and whatever other junk we want.